gnucash/import-export/import-main-matcher.c

```diff
@@ -1014,12 +1016,12 @@ GNCImportMainMatcher *gnc_gen_trans_list_new (GtkWidget *parent,
     g_object_unref (G_OBJECT(builder));
     
     // Register this UI, it needs to be closed when the session is closed.
-    id = gnc_register_gui_component (IMPORT_MAIN_MATCHER_CM_CLASS,
+    info->id = gnc_register_gui_component (IMPORT_MAIN_MATCHER_CM_CLASS,
```

Comment:
Realign the parameters below this line and also remove the 'gint id' on line 956